opm/grid/cpgrid/CartesianIndexMapper.hpp

```diff
@@ -49,6 +49,12 @@ namespace Dune
             return grid_.globalCell().size();
         }
 
+        int compressedLevelZeroSize() const
+        {
+            
+            return (*grid_.chooseData()[0]).global_cell_.size();
```

Comment:
As we want to get the number codim-0 elements on that level, could we use the more expressive
```
(*grid_.chooseData()[0]).size(0);
```
Then we would also not need to access private members and could delete the friend declaration in CpGridData

Reply:
I think global_cell_ is related to active cells too. Probably, under our assumptions for supporting LGRs, they coincide now, but might not be the case for the general setting. 

CpGridData::size( /*codim*/ 0) returns cell_to_face_.size() (which, if I get it correctly, is the total amount of cells, no matter if they are active or inactive). 

global_cell_.size() should be total amount of active cells. 

Under our assumptions, all cells on level zero, on each Lgr, and  on the leaf grid view, are active. So we can do your suggestion now.